error-reporting/api/ErrorReportingSample/ErrorReportingSample.cs

```diff
@@ -56,16 +56,31 @@ private static void report(Exception e)
             Environment.GetEnvironmentVariable("GOOGLE_PROJECT_ID"));
 
         // Add a service context to the report. For more details see:
-        // https://cloud.google.com/error-reporting/reference/rest/v1beta1/projects.events#ServiceContext
-        ServiceContext serviceContext = new ServiceContext()
+        // https://cloud.google.com/error-reporting/reference/rest/v1beta1/ServiceContext
+        var assemblyName = System.Reflection.Assembly.GetExecutingAssembly().GetName();
+        var serviceContext = new ServiceContext()
```

Comment:
I'm not certain you'd want this to be used as the service context. This is the assembly that contains the code currently executing, for instance, if your organization has a library for logging, then this will be that library's assembly.

You might want to look at [GetEntryAssembly](https://learn.microsoft.com/en-us/dotnet/api/system.reflection.assembly.getentryassembly?view=net-7.0) instead?

@jskeet In case you have other ideas.

Reply:
Replaced. :+1: